develop/plone/forms/wysiwyg.rst

```diff
@@ -37,8 +37,13 @@ If you need to put a `<script>` tag on your content text in TinyMCE you can disa
 
     tal:define="dummy python:checkPermission('Modify portal content', context) and request.RESPONSE.setHeader('X-XSS-Protection', '0');"
 
+**Step 3:** (Plone 5 only) Add script tag to the list of `extended_valid_elements` of TinyMCE. Go to the Control Panel, TinyMCE settings, Advanced tab. Add to the Other settings field::
+
```

Comment:
I guess we can remove '(Plone 5 only)' ? Because it will go only into the docs from 5.0 on, meaning currenltu it will go into the branches 5.0 and 5.1 of the docs.